world_population_analysis.yaml

```diff
@@ -0,0 +1,18 @@
+stages:
+  - name: worldpopulation
+    scheduler:
+      scheduler_type: 'singlestep-stage'
+      parameters:
+        outputdir: '{workdir}'
+        outputfile: '{workdir}/world_population_analysis.html'
+      step:
+        process:
+          process_type: 'string-interpolated-cmd'
+          cmd: 'jupyter nbconvert --output-dir="{outputdir}" world_population_analysis.ipynb'
```

Comment:
Yes, it's more instructive to keep it full.  (Especially in the optics of yadage, where the container usually defines environment only, and users define steps to run explicitly on top of that.)